### DISCUSSION TITLE
Input device from the pc as source

### DISCUSSION THREAD

**Discussion by Janoo1606** [jagrosh/MusicBot#839] [Questions]
Hello,
is it possible to get the music directly from the pc?
So you can select a input device as mic or audio input from the pc as a source for the musicbot.

Best regards

**Comment by MichailiK** [comment]
It is not possible, and doesn't really fit into a (cross-platform) Discord music bot anyway. You can find a list of supported sources [here](https://github.com/jagrosh/MusicBot#supported-sources-and-formats).  Consider using alternatives like [Voicemeeter](https://vb-audio.com/Voicemeeter/index.htm), which allow you to output multiple audio sources to a single, virtual microphone/source.
